os/centralized-logging.md

```diff
@@ -0,0 +1,138 @@
+# Centralized Logging
+
+Systmed Journald allows to transmit logs to central storage. Basic elements are:
+
+* systemd-journal-remote - collects logs from remote hosts (server)
+* systemd-journal-upload - forwards logs into centralized storage (client)
+
+systemd-journal-remote supports two kins of sources:
+
+* active - requests and pulls the data
+* passive - waits for a connection and then receives events pushed by the client side
+
+In this documentation will will describe passive method as more common one (comparing to [`rsyslogd(8)`](http://linux.die.net/man/8/rsyslogd) or [`syslog-ng`](http://linux.die.net/man/8/syslog-ng) centralized logging) and we will not use encryption in this example.
```

Comment:
This document describes the *passive* method, which may be compared to the centralized logging provided by [`rsyslogd(8)`](http://linux.die.net/man/8/rsyslogd) or [`syslog-ng(8)`](http://linux.die.net/man/8/syslog-ng). This introductory example will not cover encryption of the log stream.